docs/docs/using-gatsby-without-graphql.md

```diff
@@ -85,7 +85,7 @@ Using Gatsby's data layer provides the following benefits:
 - Improves performance by removing data bloat — GraphQL is a big part of why Gatsby is so fast as it enables lazy-loading the exact data in the exact form each view needs
 - Enables you to take advantage of hot reloading when developing; For example, in this post's example "Pokémon" site, if you wanted to add a "see other pokémon" section to the pokémon detail view, you would need to change your `gatsby-node.js` to pass all pokémon to the page, and restart the dev server. In contrast, when using queries, you can add a query and it will hot reload.
 
-> Learn more about [GraphQL in Gatsby](/docs/querying-with-graphql/).
+> Learn more about [GraphQL in Gatsby](/docs/graphql/).
```

Comment:
This actually is the file path: https://github.com/gatsbyjs/gatsby/blob/68d75f275ab2a0bf7e4453a0a0c81ac3fb296b3a/docs/docs/graphql.md

So it needs to be `/docs/docs/graphql.md`